models.py

```diff
@@ -1259,6 +1268,74 @@ class CognitionPipelineLogs(Base):
     skipped_step = Column(Boolean, default=False)
 
 
+class CognitionConsumptionLog(Base):
+    __tablename__ = Tablenames.CONSUMPTION_LOG.value
+    __table_args__ = {"schema": "cognition"}
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    organization_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"{Tablenames.ORGANIZATION.value}.id", ondelete="CASCADE"),
+        index=True,
+    )
+    project_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"cognition.{Tablenames.PROJECT.value}.id", ondelete="SET NULL"),
+        index=True,
+    )
+    strategy_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"cognition.{Tablenames.STRATEGY.value}.id", ondelete="SET NULL"),
+        index=True,
+    )
+    conversation_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(
+            f"cognition.{Tablenames.CONVERSATION.value}.id", ondelete="SET NULL"
+        ),
+        index=True,
+    )
+    message_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"cognition.{Tablenames.MESSAGE.value}.id", ondelete="SET NULL"),
+        index=True,
+    )
+    created_by = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"{Tablenames.USER.value}.id", ondelete="SET NULL"),
+        index=True,
+    )
+    created_at = Column(DateTime, default=sql.func.now())
+    complexity = Column(String)  # of type enums.StrategyComplexity.*.value
+    state = Column(String)  # of type enums.ConsumptionLogState.*.value
+    project_state = Column(String)  # of type enums.CognitionProjectState.*.value
+
+
+class CognitionConsumptionSummary(Base):
+    __tablename__ = Tablenames.CONSUMPTION_SUMMARY.value
+    __table_args__ = (
+        UniqueConstraint(
+            "organization_id", "project_id", "date", "complexity", name="unique_summary"
+        ),
+        {"schema": "cognition"},
+    )
+
+    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    organization_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"{Tablenames.ORGANIZATION.value}.id", ondelete="CASCADE"),
+        index=True,
+    )
+    project_id = Column(
+        UUID(as_uuid=True),
+        ForeignKey(f"cognition.{Tablenames.PROJECT.value}.id"),
+        index=True,
+    )
+    date = Column(Date, default=sql.func.now(), index=True)
```

Comment:
creation_date as otherwise might be confused with a "last update" field or something similar. Not sure how integrated already. Change < 1h change otherwise skip